pkg/shed/index.go

```diff
@@ -336,6 +390,26 @@ func (f Index) Iterate(fn IndexIterFunc, options *IterateOptions) (err error) {
 	return it.Error()
 }
 
+// Increment the last byte that is not 0xFF, and returned a new byte array
```

Comment:
`and return`

Reply:
also, no need to capitalize the first word. if godoc spec is used, you should start the sentence with the name of the function